### ISSUE TITLE
Laravel 5.2 support ?

### ISSUE THREAD

**Issue by IfnotFr** [pingpong-labs/widget#13]
Hi, 

I got this error with laravel 5.2 update :

```
pingpong/widget v2.1.0 requires illuminate/html 5.1.* -> no matching package found.
Conclusion: don't install laravel/framework v5.2.10
```

I dont know if the API of illuminate/html changed from 5.1 to 5.2 ...

**Comment by bradleybernard** [comment]
+1

**Comment by IfnotFr** [comment]
Thanks :)

Fixed with 4d3d4667da4b7037a0d00c42d1017e1edc0b8315
